apps/recipes-react-components/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
we should use ts solution config files like can be seen in `apps/react-18-tests-v9`